io/disk/dlpar_vscsi.py

```diff
@@ -39,6 +39,7 @@ def setUp(self):
         self.vios_pwd = self.params.get('vios_pwd', '*', default=None)
         self.session = Session(self.vios_ip, user=self.vios_user,
                                password=self.vios_pwd)
+        self.session.connect()
```

Comment:
should this also be 
```
if not self.session.connect():
         self.cancel("failed connecting to vios")
```

Reply:
@abdhaleegit its always failing for vios , if conneting also so i avoid to use if.

Reply:
if it always fails than why to add ? for vios